scripts/pylib/pytest-device-adapter/README.rst

```diff
@@ -0,0 +1,37 @@
+==============
+Device adapter
+==============
+
+Installation
+------------
+
+Installation from the source:
+
+.. code-block:: sh
+
+  pip install .
```

Comment:
I believe that it can be helpful for developers who work more with pytest tests (like me) to have it as an installed package. But if it's so confusing for you, I remove this instruction from the `README` file.